packages/loot-design/src/components/modals/SelectLinkedAccounts.js

```diff
@@ -1,254 +1,200 @@
+                        : localAccounts.find(
+                            acc => chosenAccounts[item.account_id] === acc.id,
+                          )
+                    }
+                    unlinkedAccounts={unlinkedAccounts}
+                    onSetLinkedAccount={onSetLinkedAccount}
+                  />
                 </View>
               )}
-            </View>
+            />
           </View>
-
           <View
             style={{
               flexDirection: 'row',
               justifyContent: 'flex-end',
-              marginTop: 30,
+              marginTop: 10,
             }}
           >
-            <Button style={{ marginRight: 10 }} onClick={modalProps.onClose}>
-              Cancel
-            </Button>
-            <Button primary onClick={onNext} disabled={!chosenAccounts.length}>
+            <Button
+              primary
+              onClick={onNext}
+              disabled={!Object.keys(chosenAccounts).length}
+            >
               Link accounts
             </Button>
           </View>
-        </View>
+        </>
       )}
     </Modal>
   );
 }
+
+function TableRow({
+  externalAccount,
+  linkedAccount,
+  chosenAccount,
+  unlinkedAccounts,
+  onSetLinkedAccount,
+}) {
+  const [focusedField, setFocusedField] = useState(null);
+
+  const chosenAccountOption = chosenAccount && {
+    value: chosenAccount.id,
+    label: chosenAccount.name,
+  };
+
+  const availableAccountOptions = [
+    ...unlinkedAccounts.map(acct => ({
+      value: acct.id,
+      label: acct.name,
+    })),
+    chosenAccount?.id !== addAccountOption.value && chosenAccountOption,
+    addAccountOption,
+  ].filter(Boolean);
+
+  return (
+    <Row style={{ backgroundColor: 'white' }}>
+      <Field width={200}>{externalAccount.name}</Field>
+      <Field
+        width="flex"
+        truncate={focusedField !== 'account'}
+        onClick={() => !linkedAccount && setFocusedField('account')}
+      >
+        {focusedField === 'account' ? (
+          <Autocomplete
+            autoFocus
+            options={availableAccountOptions}
+            onSelect={value => {
+              onSetLinkedAccount(externalAccount, value);
+            }}
+            onBlur={() => setFocusedField(null)}
+            value={chosenAccountOption}
+          />
+        ) : (
+          linkedAccount?.name || chosenAccount?.name
+        )}
+      </Field>
+      <Field width="flex">
+        {!linkedAccount &&
+          (chosenAccount ? (
+            <Button
+              onClick={() => {
+                onSetLinkedAccount(externalAccount, null);
+              }}
+              style={{ float: 'right' }}
+            >
+              Remove bank-sync
+            </Button>
+          ) : (
+            <Button
+              primary
+              onClick={() => {
+                setFocusedField('account');
+              }}
+              style={{ float: 'right' }}
+            >
+              Setup bank-sync
```

Comment:
Start syncing?

Reply:
Hmm.. so basically there are 2x actions: addition and removal. The labels of these actions should be in sync (i.e. we should not use "add bank-sync" and "stop syncing"; instead: start/stop in both or add/remove in both).

We should not use "link/unlink" because the linking actually happens only after clicking the "link" button at the bottom on the modal.

Other options:
- start/stop
- add/remove
- setup/teardown (?)
- connect/disconnect account
- ?

Personally I'm leaning now more towards "connect/disconnect account". WDYT?